generators/app/templates/src/StarterKit/StarterKit.csproj

```diff
@@ -45,6 +43,16 @@
     <PackageReference Include="Digipolis.Serilog.Correlation" Version="4.0.0" />
     <PackageReference Include="Digipolis.Serilog.Message" Version="2.0.0" />
     <PackageReference Include="Digipolis.ServiceAgents" Version="7.1.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.9" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="3.1.9" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.9" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.9">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="MongoDB.Bson" Version="2.11.4" />
+    <PackageReference Include="MongoDB.Driver" Version="2.11.4" />
```

Comment:
Wordt er bij yeoman-generatie rekening gehouden dat de Mongo-references niet altijd ingeladen moeten worden? Voorheen gebeurde dit door placeholder-tags (bv. <!-- dataaccess-package -->)

Reply:
Dat had ik inderdaad niet meegenomen. Het is nu aangepast. Ik werk nu met een start en end tag die alles daartussen verwijderd afhankelijk van je gekozen DB provider. Zo kan je in de starterkit zelf alle references behouden en wordt alles netjes opgeschoont bij de generatie.